src/tye_lab_to_nwb/neurotensin_valence/neurotensin_valencenwbconverter.py

```diff
@@ -16,3 +25,37 @@ def get_metadata_schema(self) -> dict:
         # Manual override to allow additional properties for pose estimation metadata
         metadata_schema["additionalProperties"] = True
         return metadata_schema
+
+    def get_metadata(self):
+        """Auto-fill as much of the metadata as possible. Must comply with metadata schema."""
+        metadata = super().get_metadata()
+        start_times = []
+        for interface in self.data_interface_objects.values():
+            interface_metadata = interface.get_metadata()
+            if "NWBFile" not in interface_metadata:
+                continue
+            if "session_start_time" in interface_metadata["NWBFile"]:
+                start_times.append(interface_metadata["NWBFile"]["session_start_time"])
+
+        # Use the earliest session_start_time
+        metadata["NWBFile"].update(session_start_time=min(start_times))
+
+        return metadata
+
+    def run_conversion(
+        self,
+        nwbfile_path: Optional[str] = None,
+        nwbfile: Optional[NWBFile] = None,
+        metadata: Optional[dict] = None,
+        overwrite: bool = False,
+        conversion_options: Optional[dict] = None,
+    ):
+        recording_interface = self.data_interface_objects["Recording"]
+        # manually override t_start
+        recording_interface.recording_extractor._recording_segments[0].t_start = None
```

Comment:
The value of the first timestamp for all channels is `465704704`. When divided by the sampling frequency of `30000.0 `we get `15523.490133333333` value as starting time. Here I override t_start to be None instead of `15523.490133333333`, this way the `starting_time` of the `ElectricalSeries` in `nwbfile.acquisition['ElectricalSeries']` is `0.0`